### PR TITLE
Propose @Yisaer as committer of sig-exec

### PR DESCRIPTION
**Pull request by SunRunAway**
Since @Yisaer has contributed a lot to sig-exec, according to sig-exec [roles-and-organization-management.md#promotion](https://github.com/pingcap/community/blob/master/special-interest-groups/sig-exec/roles-and-organization-management.md#from-reviewer-to-committer), we would like to promote him to our new committer, and his detailed contribution is shown below:

1. contribute [20+](https://github.com/pingcap/tidb/pulls?q=is%3Apr+author%3Ayisaer+label%3Astatus%2Fcan-merge+label%3Asig%2Fexecution) PRs to the executor,
2. support the [global memory tracker](https://github.com/pingcap/tidb/pull/16777) (a medium task)
4. support the ratelimit action to solve the [table reader oom problem](https://github.com/pingcap/tidb/issues/16104) (a hard task)
5. help other contributors and review [25+](https://github.com/pingcap/tidb/pulls?q=is%3Apr+reviewed-by%3Ayisaer+-author%3Ayisaer) PRs.

Thanks for his contribution!!

### CODE REVIEW

**Comment by SunRunAway** [comment]
@XuHuaiyu @qw4990 @winkyao PTAL

**Comment by XuHuaiyu** [comment]
LGTM

File: special-interest-groups/sig-exec/membership.md [hunk 1]
```diff
@@ -29,7 +29,7 @@
 - [hanfei1991](https://github.com/hanfei1991)
 - [mmyj](https://github.com/mmyj)
 - [b41sh](https://github.com/b41sh)
-
+- [@Yisaer](https://github.com/Yisaer)
```
Comment (Yisaer) [inline special-interest-groups/sig-exec/membership.md:32]:
```suggestion
- [Yisaer](https://github.com/Yisaer)
```

**Review by Yisaer — COMMENTED** [review]
Rest LGTM

**Comment by hi-rustin** [comment]
Migrate to json for sig-execution membership(#354)

**Comment by SunRunAway** [comment]
see https://github.com/pingcap/community/pull/355

**Pull request closed without merging**